.rubocop.yml

```diff
@@ -0,0 +1,52 @@
+AllCops:
+  EnabledByDefault: true
+  Exclude:
+    - "README.md"
+    - "Guardfile"
+    - "Rakefile"
+
+  DisplayCopNames: true
+
+Layout/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Max: 20
+Metrics/AbcSize:
+  Max: 50
+Metrics/ClassLength:
+  Max: 150
+Metrics/BlockLength:
+  ExcludedMethods: ['describe']
+  Max: 30
+
+
+Style/Documentation:
+  Enabled: false
+Style/ClassAndModuleChildren:
+  Enabled: false
+Style/EachForSimpleLoop:
+  Enabled: false
+Style/AndOr:
+  Enabled: false
+Style/DefWithParentheses:
+  Enabled: false
+Style/FrozenStringLiteralComment:
+  EnforcedStyle: never
+
+Layout/HashAlignment:
+  EnforcedColonStyle: key
+Layout/ExtraSpacing:
+  AllowForAlignment: false
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Style/Copyright:
+  Enabled: false
+Style/DocumentationMethod:
+  Enabled: false
+Style/InlineComment:
+  Enabled: false
+Style/MissingElse:
+  Enabled: false
+Style/MethodCalledOnDoEndBlock:
+  Enabled: false
```

Comment:
Why do you need to make changes to the rubcop.yml file? You should not alter it. If you need to disable any rules, you have to do so by inline comments that you have done already. Please read the instruction in [here](https://github.com/microverseinc/linters-config/tree/master/ruby).

Reply:
The following cops were added to RuboCop, but are not configured. Please set Enabled to either true or false in your .rubocop.yml file:

Lint/RaiseException (0.81)
Lint/StructNewOverride (0.81)
Style/HashEachMethods (0.80)
Style/HashTransformKeys (0.80)
Style/HashTransformValues (0.80)
that's the linter error I get after replacing my rubocop file with the default one
if you're requesting me to to update it and rubocop if complaining when I try using inline
what do you suggest I do?